bluetooth_mesh/models/base.py

```diff
@@ -515,3 +518,13 @@ async def bind(self, app_key_index: int) -> "ModelBindStatus":
         """
 
         return await self.element.application.bind_app_key(app_key_index, model=self)
+
+
+class ModelConfig(NamedTuple):
+    """
+    Model Configuration class for mesh models.
+    """
+
+    bindings: List[int] = []
+    publication_period: timedelta = None
+    subscriptions: Set[Union[int, UUID]] = set()
```

Comment:
Careful. I think non-scalar default values should be handled using a factory function. Please consult the docs.

```
>>> from typing import NamedTuple
>>> class Foo(NamedTuple):
...   some_list : list = []
... 
>>> foo = Foo()
>>> foo.some_list
[]
>>> foo.some_list.append(1)
>>> bar = Foo()
>>> bar.some_list
[1]
```